tests/gpflow/models/test_svgp_posterior.py

```diff
@@ -0,0 +1,77 @@
+from itertools import product
+from typing import Tuple
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+import gpflow
+from gpflow.models.svgp import NewSVGP, OldSVGP
+
+input_dim = 7
```

Comment:
I'd make it a private module (global) variable.

```suggestion
input_dim = 7
```

```suggestion
__input_dim = 7
```